### PR TITLE
Adding Glue catalog configuration in Iceberg document

### PR DESCRIPTION
**Pull request by agrawalreetika**
Test plan - (Please fill in how you tested your changes)

Please make sure your submission complies with our [Development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [Formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), and [Commit Message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests) guidelines. Don't forget to follow our [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution) for any code copied from other projects.

Fill in the release notes towards the bottom of the PR description.
See [Release Notes Guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) for details.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by ajaygeorge — APPROVED** [review]
LGTM

**Review by ChunxuTang — CHANGES_REQUESTED** [review]
Hi @agrawalreetika,

Thanks for your contribution!
To my knowledge, we don't support the Glue catalog in the Iceberg connector yet. For now, we support Hive/Hadoop/Nessie catalogs. (https://prestodb.io/docs/current/connector/iceberg.html#iceberg-catalog-type)

@beinan Feel free to correct me If I'm wrong.

**Comment by agrawalreetika** [comment]
> Hi @agrawalreetika,
> 
> Thanks for your contribution! To my knowledge, we don't support the Glue catalog in the Iceberg connector yet. For now, we support Hive/Hadoop/Nessie catalogs. (https://prestodb.io/docs/current/connector/iceberg.html#iceberg-catalog-type)
> 
> @beinan Feel free to correct me If I'm wrong.

Hi @ChunxuTang,
We can connect to Glue as metastore via iceberg connector by using `iceberg.catalog.type=hive`. My understanding here is glue as catalog meaning using it as metastore, please correct me if I am missing something here.

File: presto-docs/src/main/sphinx/connector/iceberg.rst [hunk 1]
```diff
@@ -18,12 +18,32 @@ To configure the Iceberg connector, create a catalog properties file
 ``etc/catalog/iceberg.properties`` with the following contents,
 replacing the properties as appropriate:
 
+Hive Metastore catalog
+^^^^^^^^^^^^^^^^^^^^^^
+
+Iceberg connector supports the same configuration for
+`HMS <https://prestodb.io/docs/current/connector/hive.html#metastore-configuration-properties>`_
+as Hive connector.
+
```
Comment (ChunxuTang) [inline presto-docs/src/main/sphinx/connector/iceberg.rst:27]: We already have a section to discuss the catalog types supported by the iceberg connector. Would you like to move this to that section (https://prestodb.io/docs/current/connector/iceberg.html#iceberg-catalog-type)?

File: presto-docs/src/main/sphinx/connector/iceberg.rst [hunk 2]
```diff
@@ -18,12 +18,32 @@ To configure the Iceberg connector, create a catalog properties file
...
+Iceberg connector supports the same configuration for
+`Glue <https://prestodb.io/docs/current/connector/hive.html#aws-glue-catalog-configuration-properties>`_
+as Hive connector.
+
+.. code-block:: none
+
+    connector.name=iceberg
+    hive.metastore=glue
+    iceberg.catalog.type=hive
+
```
Comment (ChunxuTang) [inline presto-docs/src/main/sphinx/connector/iceberg.rst:46]: Same as above. Would you move this to the catalog type section? As the Glue catalog is still the Hive metastore catalog, maybe you can just mention it there with the appropriate configurations. Your call.

**Review by ChunxuTang — COMMENTED** [review]
Hi @agrawalreetika,

Thanks for your clarification! As we don't have AWS Glue tests in the Iceberg connector, I'm not sure whether it works well though it should. Do you happen to know any successful usages of the Glue catalog in the Iceberg connector?

BTW, I left some minor suggestions in the PR. Feel free to have a look!

**Comment by agrawalreetika** [comment]
> Hi @agrawalreetika,
> 
> Thanks for your clarification! As we don't have AWS Glue tests in the Iceberg connector, I'm not sure whether it works well though it should. Do you happen to know any successful usages of the Glue catalog in the Iceberg connector?
> 
> BTW, I left some minor suggestions in the PR. Feel free to have a look!

Thanks, @ChunxuTang for your review. I will update PR accordingly.
I had tried out example queries given on the connector page by connecting to Glue as Metastore. Those worked fine!

File: presto-docs/src/main/sphinx/connector/iceberg.rst (hunk 1 above)
Comment (agrawalreetika) [reply presto-docs/src/main/sphinx/connector/iceberg.rst:27]: Hi @ChunxuTang, I was thinking wouldn't it be better to have 2 sections for catalog config just like https://prestodb.io/docs/current/connector/iceberg.html#nessie-catalog ?

Comment (ChunxuTang) [reply presto-docs/src/main/sphinx/connector/iceberg.rst:27]: That's a nice idea! Sure. Feel free to put the catalog configs together with separate sections.

**Comment by ChunxuTang** [comment]
> > Hi @agrawalreetika,
> > Thanks for your clarification! As we don't have AWS Glue tests in the Iceberg connector, I'm not sure whether it works well though it should. Do you happen to know any successful usages of the Glue catalog in the Iceberg connector?
> > BTW, I left some minor suggestions in the PR. Feel free to have a look!
> 
> Thanks, @ChunxuTang for your review. I will update PR accordingly. I had tried out example queries given on the connector page by connecting to Glue as Metastore. Those worked fine!

@agrawalreetika Thanks for your work! It's nice that queries could work with AWS Glue. Feel free to ping me when you update the PR. Maybe we just need to group Hive metastore, AWS Glue, and other catalog types together with separate sections for configs.

**Comment by agrawalreetika** [comment]
@ChunxuTang  I have updated the PR, please check.

**Review by ChunxuTang — APPROVED** [review]
LGTM. @agrawalreetika Thanks for your nice work!

**Pull request merged**